earth_enterprise/rpms/opengee-common/post-uninstall.sh

```diff
@@ -22,18 +22,23 @@ BININSTALLROOTDIR="/etc/init.d"
 
 #------------------------------------------------------------------------------
 # Group names:
-GEGROUP_NAME=$(cat "$BININSTALLROOTDIR/gevars.sh" | grep GEGROUP | cut  -d'=' -f2)
-
 
 remove_users_groups()
```

Comment:
Suggestion: Consider replacing this script with just printing a message that the system administrator may want to remove `gegroup`, if there are now files or directories using it.